docs/gravitino-mcp-server.md

```diff
@@ -53,7 +53,7 @@ Or start a HTTP MCP server by `uv run mcp_server --metalake test --uri http://12
 
 ### Docker instructions
 
-You could start Gravitino MCP server by docker image, `docker run -p 8000:8000 apache/gravitino-mcp-server:latest --metalake test --transport http --mcp-url http://0.0.0.0:8000/mcp`. Please note that the MCP server in docker container doesn't support `stdio` transport mode.
+You could start Gravitino MCP server by docker image, `docker run -p 8000:8000 --network=host apache/gravitino-mcp-server:latest --metalake test --transport http --mcp-url http://0.0.0.0:8000/mcp --uri=http://127.0.0.1:8090`. Please note that the MCP server in docker container doesn't support `stdio` transport mode.
```

Comment:
capitabliz D for Docker. Also, I was thinking that is it much clearer to use `--gravitino-url` than `--uri`, WDYT?

Reply:
As both xx connectors and fileset use `uri` to address Gravitino server,  how about using `gravitino-uri`?